connectors/src/connectors/slack/bot.ts

```diff
@@ -42,8 +42,9 @@ export async function botAnswerMessageWithErrorHandling(
   slackChannel: string,
   slackUserId: string,
   slackMessageTs: string,
-  slackThreadTs: string | null
-): Promise<Result<AgentGenerationSuccessEvent, Error>> {
+  slackThreadTs: string | null,
+  mentionOverride: string[]
```

Comment:
Why this name? It's never an "override" per say as this can only be filled it there was no mention in the content of the message initially right? 

Also, why an array over string or null? Can we have multiple value in here?